api/routes/index.js

```diff
@@ -2,7 +2,9 @@ const express = require("express");
 const router = express.Router();
 
 const userRouter = require("./userRouter");
+const cartRouter = require("./cartRouter");
```

Comment:
```suggestion
const userRouter = require("./userRouter");
const cartRouter = require("./cartRouter");

const router = express.Router();
```
모듈을 불러오는 코드가 먼저 작성되는게 일반적입니다.